tri.html

```diff
@@ -1,58 +1,61 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Triangle Game</title>
-    <link rel="stylesheet" type="text/css" href="tri.css">
-  </head>
-  <body>
-      
-    <h1>The <span id="titleDisplay">Triangle</span> Game</h1>
-
-    <div id="stripe">
-      <button class="btn" id="reset"></button>
-    </div>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Triangle Game</title>
+  <link rel="stylesheet" type="text/css" href="tri.css">
+</head>
+
+<body>
+
+  <h1>The <span id="titleDisplay">Triangle</span> Game</h1>
+
+  <div id="stripe">
+    <button class="btn" id="reset"></button>
+  </div>
 
 
-    <div class="triangle">
-  
+  <div class="triangle">
+
     <div class="wrapper">
 
-    <div class="container1a" ></div>
+      <div class="container1a"></div>
 
-    <div class="container1" >
-         <div class="circle1" id="one">1</div>
-    </div>
-   
-    <div class="container2" >
-         <div class="circle2" id="two">2</div>
-         <div class="circle2" id="three">3</div>
-    </div>
-     
-    <div class="container3" >
-        <div class="circle3" id="four">4</div>
-        <div class="circle3" id="five">5</div>
-        <div class="circle3" id="six">6</div>
+      <div class="container1">
+        <div class="circle" id="one">1</div>
       </div>
-  
-      <div class="container4" >
-          <div class="circle4" id="seven">7</div>
-          <div class="circle4" id="eight">8</div>
-          <div class="circle4" id="nine">9</div>
-          <div class="circle4" id="ten">10</div>
+
+      <div class="container2">
+        <div class="circle" id="two" onclick="selectCircle(this)">2</div>
+        <div class="circle" id="three" onclick="selectCircle(this)">3</div>
       </div>
 
-      <div class="container5" >
-            <div class="circle5" id="eleven">11</div>
-            <div class="circle5" id="twelve">12</div>
-            <div class="circle5" id="thirteen">13</div>
-            <div class="circle5" id="fourteen">14</div>
-            <div class="circle5" id="fifteen">15</div>
-     </div> 
+      <div class="container3">
+        <div class="circle" id="four" onclick="selectCircle(this)">4</div>
+        <div class="circle" id="five" onclick="selectCircle(this)">5</div>
+        <div class="circle" id="six" onclick="selectCircle(this)">6</div>
+      </div>
+
+      <div class="container4">
+        <div class="circle" id="seven" onclick="selectCircle(this)">7</div>
+        <div class="circle" id="eight" onclick="selectCircle(this)">8</div>
+        <div class="circle" id="nine" onclick="selectCircle(this)">9</div>
+        <div class="circle" id="ten" onclick="selectCircle(this)">10</div>
+      </div>
+
+      <div class="container5">
```

Comment:
I'd change `container[N]` to a more descriptive name like `row`. You can probably give each row the same class name so you can style them all together, then use an Id, numbering, or data attribute to style the rows individually.
```suggestion
      <div class="row" id="row-5">
```

Reply:
Completed. Thanks.